### PR TITLE
fix: compliance tooltip on project card

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: APP-454

Fix hectare compliance tooltip.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
Preview URL: https://deploy-preview-2541--regen-marketplace.netlify.app/

**Comment by erikalogie** [comment]
LGTM!

File: web-marketplace/src/pages/Projects/AllProjects/AllProjects.TerrasosCredits.tsx [hunk 1]
```diff
@@ -53,7 +54,7 @@ export const TerrasosCredits = ({
             placement="top"
             title={_(COMPLIANCE_MARKET_TOOLTIP)}
           >
-            <HectaresBadge className="ml-[3px]" />
```
Comment (blushi) [inline web-marketplace/src/pages/Projects/AllProjects/AllProjects.TerrasosCredits.tsx:56]: can we get rid of `HectaresBadge`?
looks like it's only used in a story now

File: web-marketplace/src/pages/Projects/AllProjects/AllProjects.TerrasosCredits.tsx [hunk 2]
```diff
@@ -53,7 +54,7 @@ export const TerrasosCredits = ({
...
+            <img src={hectaresLogo} alt="hectares" className="ml-[3px]" />
```
Comment (blushi) [inline web-marketplace/src/pages/Projects/AllProjects/AllProjects.TerrasosCredits.tsx:57]: should we i18n `alt`?

**Review by blushi — APPROVED** [review]
Looks good, pending a couple of nits

**Pull request merged**